homeassistant/components/heos/.translations/ca.json

```diff
@@ -9,12 +9,12 @@
         "step": {
             "user": {
                 "data": {
-                    "access_token": "Amfitri\u00f3"
+                    "host": "Amfitri\u00f3"
                 },
                 "description": "Introdueix el nom d'amfitri\u00f3 o l'adre\u00e7a IP d'un dispositiu Heos (preferiblement un connectat a la xarxa per cable).",
                 "title": "Connexi\u00f3 amb Heos"
             }
         },
-        "title": "Heos"
```

Comment:
This won't help, these files are auto-generated. Will need to be fixed in lokalise.co

Reply:
OK I'll revert those changes.